src/styles/index.scss

```diff
@@ -1,3 +1,35 @@
+@import './utils/variables';
+@import './blocks/stars';
+@import './blocks/card';
+@import './blocks/header';
+
 body {
   margin: 0;
 }
+
+.container {
+  display: grid;
+  justify-content: center;
+  grid-template-columns: repeat(1, 200px);
+  padding: 50px 40px;
+  gap: 46px 48px;
```

Comment:
The values `200px`, `50px`, `40px`, `46px`, and `48px` are used multiple times. According to the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), it's recommended to use variables for repeated values to make the code more maintainable and readable. Consider adding them to `variables.scss` and then importing them here.